src/main/java/at/ac/fhcampuswien/snake/board/ScoreBoard.java

```diff
@@ -0,0 +1,62 @@
+package at.ac.fhcampuswien.snake.board;
+
+import javafx.application.Platform;
+import javafx.geometry.VPos;
+import javafx.scene.canvas.Canvas;
+import javafx.scene.canvas.GraphicsContext;
+import javafx.scene.paint.Color;
+import javafx.scene.text.Font;
+import javafx.scene.text.Text;
+import javafx.scene.text.TextAlignment;
+
+import static at.ac.fhcampuswien.snake.util.Constants.*;
+
+public class ScoreBoard {
+    private final GraphicsContext gc;
+
+    public ScoreBoard(Canvas scoreBoardCanvas) {
+        this.gc = scoreBoardCanvas.getGraphicsContext2D();
+    }
+
+    public void drawScoreBoard(int score) {
+        this.gc.setFill(Color.web("4a148c"));
+        this.gc.fillRect(0, 0, SCOREBOARD_WIDTH, SCOREBOARD_HEIGHT);
+        gc.setTextAlign(TextAlignment.RIGHT);
+        gc.setTextBaseline(VPos.CENTER);
+        this.gc.setFill(Color.WHITE);
+        this.gc.setFont(Font.font("Courier", OBJECT_SIZE_MEDIUM));
+        this.gc.fillText("Score: " + score, SCOREBOARD_WIDTH - 7, SCOREBOARD_HEIGHT / 2);
```

Comment:
Why using - 7 and divided by 2? 

Do not use magic numbers. If needed describe the code